Fail to install mongodb driver

Hello, thanks for the hard work! Unfortunately, I encountered the following error with your setup code.

```
CMake Error at src/bsoncxx/CMakeLists.txt:113 (find_package):
  Could not find a configuration file for package "libbson-1.0" that is
  compatible with requested version "1.13.0".

  The following configuration files were considered but not accepted:

    /usr/lib/x86_64-linux-gnu/cmake/libbson-1.0/libbson-1.0-config.cmake, version: 1.9.2
```
I am using Ubuntu 18.04 and Visual Studio Code for your information. CMake version 3.25.1
Any suggestion is highly appreciated.

--- a/setup-mongocxx
+++ b/setup-mongocxx
@@ -4,11 +4,10 @@
 # https://developer.mongodb.com/community/forums/t/problems-trying-to-compile-mongocxx/2084
 function install_mongocxx_dependencies() {
   TOP_DIR=$(pwd)
-  sudo apt-get install libmongoc-1.0-0 libbson-1.0 cmake libssl-dev libsasl2-dev zlib1g-dev
-
-  wget https://github.com/mongodb/mongo-c-driver/releases/download/1.17.0/mongo-c-driver-1.17.0.tar.gz
-  tar xzf mongo-c-driver-1.17.0.tar.gz
-  cd mongo-c-driver-1.17.0
+  sudo apt-get install cmake libmongoc-1.0-0 libbson-1.0 cmake libssl-dev libsasl2-dev zlib1g-dev
+  wget  https://github.com/mongodb/mongo-c-driver/releases/download/1.23.2/mongo-c-driver-1.23.2.tar.gz
+  tar xzf mongo-c-driver-1.23.2.tar.gz
+  cd mongo-c-driver-1.23.2
   mkdir cmake-build
   cd cmake-build
   cmake -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF ..
@@ -26,7 +25,7 @@ function install_mongocxx_dependencies() {
   sudo make
   sudo make install
   cd $TOP_DIR
-  rm -rf mongo-c-driver-1.17.0*
+  rm -rf mongo-c-driver-1.23.2*
 }
 
 function main() {

